### PR TITLE
[import wizard 2] add templates page

### PR DESCRIPTION
**Pull request by chejennifer**
- add template selection page
- add mapping page that will render a mapping section depending on the templateId
    - This is just the skeleton and does not have logic implemented yet
- will add navigation and integration into the tool in future PR
Screen Shot 2022-08-25 at 11 01 14 AM
Screen Shot 2022-08-25 at 11 23 04 AM

### CODE REVIEW

File: static/css/import_wizard2.scss [hunk 1]
```diff
@@ -14,4 +14,110 @@
...
\ No newline at end of file
+@import "base";
+
+table {
+  display: block;
+  width: fit-content;
+  max-width: 100%;
+  overflow: auto;
+  font-size: 0.9rem;
+  max-height: 50vh;
```
Comment (juliawu) [inline static/css/import_wizard2.scss:25]: Style suggestion: [Google's CSS Style Guide](https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order) recommends alphabetizing declarations.

File: static/js/import_wizard2/components/mapping_page.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  if (props.csvData && props.csvData.rawCsvFile) {
+    fileName = props.csvData.rawCsvFile.name;
+  } else if (props.csvData && props.csvData.rawCsvUrl) {
+    fileName = props.csvData.rawCsvUrl;
+  }
+  const MappingSectionComponent =
+    TEMPLATE_MAPPING_SECTION_COMPONENTS[props.selectedTemplate];
+  return (
+    <>
+      <h2>Step 3: Refine table format</h2>
```
Comment (juliawu) [inline static/js/import_wizard2/components/mapping_page.tsx:50]: Nit: Perhaps there's a more descriptive name we can use for this step? "Refine table format" is a little unclear to me as a naive user.

File: static/js/import_wizard2/templates.tsx [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+        <td>...</td>
+      </tr>
+    </tbody>
+  </table>
+);
+
+const MULTI_VAR_MULTI_DATE_COL_EXPLANATION = (
+  <div>
+    This file containes data about the variables &quot;Female Population&quot;
+    and &quot;Male Population&quot;, with data about a single place adn date on
```
Comment (juliawu) [inline static/js/import_wizard2/templates.tsx:128]: >with data about a single place adn date on

adn -> and

**Review by juliawu — APPROVED** [review]
LGTM, @beets may have more comments

File: static/js/import_wizard2/components/mapping_page.tsx (hunk 2 above)
Comment (chejennifer) [reply static/js/import_wizard2/components/mapping_page.tsx:50]: ah yes, so in the mock, this title was just a placeholder & actual title tbd, I'll add a comment in the code

File: static/css/import_wizard2.scss (hunk 1 above)
Comment (chejennifer) [reply static/css/import_wizard2.scss]: done

File: static/js/import_wizard2/templates.tsx (hunk 3 above)
Comment (chejennifer) [reply static/js/import_wizard2/templates.tsx]: done

**Comment by chejennifer** [comment]
thanks for the review! Will merge now & if there are additional comments from @beets, I can open a new PR to address them

**Pull request merged**